base_local_planner/src/speed_limiters/obstacle_speed_limiter.cpp

```diff
@@ -100,8 +125,20 @@ bool ObstacleSpeedLimiter::calculateLimits(double& max_allowed_linear_vel, doubl
       max_allowed_angular_vel = angular_speed;
     }
   }
-  ROS_DEBUG_THROTTLE(0.2, "Setting max speed to %f, %f", max_allowed_linear_vel, max_allowed_angular_vel);
+  if(distance_limiting == 10){
```

Comment:
Why 10?  Also, direct float comparison can have issues sometimes.  Better to do a >= or something like that.

Reply:
just an arbitrary large distance away from the robot instead of using std::numeric_limits::max